### DISCUSSION TITLE
Meta面臨8起青少年相關訴訟，造成飲食失調、嚴重憂鬱與睡眠障礙，母親指控FB、IG與11歲女兒的自殺有關

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#432] [科技新知]
祖克柏的公司meta雖為我們的生活帶來許多方便，fb、ig也讓人與人之間更容易交流，但它也引發很多問題，如文中這些對青少年的傷害。這可能是祖克柏在積極拓展公司甚至最近也將股票改名meta朝更多元方向發展時同時也更需注意的問題。

資料來源：https://www.thenewslens.com/article/168126?_gl=1*1cx7lno*_ga*MTJWNUY1YjJEU1BRaGZueWRLRUN4X1U3bHF0SEhPT1pBa2NyVlk2Szl5SmxMZnJ0UFJpTkxsOVU1T2xOSElGMg..

**Comment by YuHengChen54** [comment]
不過感覺這種問題家長自己也要管好，不然不管再怎麼限制都還是防不勝防

**Comment by brian431** [comment]
> 祖克柏的公司meta雖為我們的生活帶來許多方便，fb、ig也讓人與人之間更容易交流，但它也引發很多問題，如文中這些對青少年的傷害。這可能是祖克柏在積極拓展公司甚至最近也將股票改名meta朝更多元方向發展時同時也更需注意的問題。
> 
> 
> 
> 資料來源：https://www.thenewslens.com/article/168126?_gl=1*1cx7lno*_ga*MTJWNUY1YjJEU1BRaGZueWRLRUN4X1U3bHF0SEhPT1pBa2NyVlk2Szl5SmxMZnJ0UFJpTkxsOVU1T2xOSElGMg..

我認為限制這些社交平台並不是解決根本問題的方法，再怎麼限制都還是有突破規則的方法。對於小孩的問題父母的教育方式還是最重要的。

**Comment by lx3211** [comment]
近期興起的小紅書也造成了許多人容貌焦慮，但學會自我調適也是非常重要的，不然將會無法在這個資訊爆炸的時代生存下去。

**Comment by xCk27x** [comment]
最主要還是對正確資訊的判斷能力
如果沒有長輩給出正確指引
很容易就會讓錯誤的價值觀進入孩童的觀念中

**Comment by williamyen0623** [comment]
新興科技的發展肯定有利有弊
人類文明必然需要前進
我認爲這類問題有需要被討論的重要性
但並不代表ig fb這些社群媒體對所有青少年群體是有害的

**Comment by BingRong1217** [comment]
社群軟體確實會造成一些青少年的心理問題

**Comment by limitx0** [comment]
社群平台推送貼文的演算法的問題吧
一直讓使用者看到讓人焦慮的貼文
不然只能讓使用者不要關注那些內容

**Comment by larry40305** [comment]
平常不陪小孩，出事了才怪別人
